test/unittests/iotools/test_bids_utils.py

```diff
@@ -182,7 +182,9 @@ def create_clinical_data(tmp_path: Path, study_name: StudyName) -> Path:
                 "Age": ["45", "50", "55", "60", "60"],
             }
         )
-        df_oasis.to_csv(clinical_path / "oasis_cross-sectional.csv", index=False)
+        df_oasis.to_csv(
```

Comment:
Shouldn't this be a call to the `to_excel` method ?

Reply:
You're right !